0.download_data/jump_dataset.csv

```diff
@@ -0,0 +1,51 @@
+source_4,2020_11_04_CPJUMP1,BR00117051,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00117051/BR00117051.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00117052,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00117052/BR00117052.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00117053,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00117053/BR00117053.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00117054,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00117054/BR00117054.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00117055,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00117055/BR00117055.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118039,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118039/BR00118039.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118040,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118040/BR00118040.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118041,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118041/BR00118041.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118042,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118042/BR00118042.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118043,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118043/BR00118043.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118044,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118044/BR00118044.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118045,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118045/BR00118045.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118046,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118046/BR00118046.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118047,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118047/BR00118047.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118048,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118048/BR00118048.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118049,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118049/BR00118049.sqlite,COMPOUND
+source_4,2020_11_04_CPJUMP1,BR00118050,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00118050/BR00118050.sqlite,COMPOUND
```

Comment:
I still see only `COMPOUND` platemap types. Shouldn't there be others (e.g. CRISPR)?

Reply:
I removed this column, I created it arbitrarily at first and realized later it is not useful.